src/commands/install.yml

```diff
@@ -7,8 +7,16 @@ parameters:
     description: "Install the Hugo extended binary?"
     type: boolean
     default: true
+  install-location:
+    description: "Location where the hugo binary should be installed."
+    type: string
+    default: "/usr/local/bin"
+
 steps:
   - detect/init
+  - restore_cache:
```

Comment:
Have you tested this? I had a discussion with Glen when I did this with the Go orb. We determined that this wasn't always faster and can get expensive for CircleCI to download from the cache.

Reply:
See the CI/CD pipelines - the effect is relatively negligible here, and isn't really worth it unless GitHub goes down/rate limits us (which has happened previously, but is relatively rare). On the fence about keeping it myself, I recall those conversations.

Reply:
They ratelimit us when we access their JSON (to determine latest version). The actual downloads themselves I don't remember getting rate limited on but I could be mistaken.

Reply:
Features was based on https://github.com/CircleCI-Public/hugo-orb/issues/28 - I don't recall them rate limiting our downloads either. Probably fine to remove and close related issue, barring network issues within jobs either one is just as fast.